libs/matdbg/src/DebugServer.cpp

```diff
@@ -410,5 +448,75 @@ const DebugServer::MaterialRecord* DebugServer::getRecord(const MaterialKey& key
     return iter == mMaterialRecords.end() ? nullptr : &iter->second;
 }
 
+bool DebugServer::handleEditCommand(const MaterialKey& key, backend::Backend(api), int shaderIndex,
```

Comment:
Did you mean `backend::Backend(api)` here?

Reply:
Yeah this is an intentional cast from int to enum.  The "api" variable is perhaps poorly named.

Reply:
Woops I was looking at the wrong codeline.  Thanks for catching this Ben.  Fixed.

Reply:
I'm surprised it compiles!

Reply:
This is c++ :p
`backend::Backend(api)`
is equal to:
`backend::Backend api`

https://youtu.be/lkgszkPnV8g?t=1795

Reply:
TIL!